pkg/cmd/server/origin/master.go

```diff
@@ -16,7 +16,7 @@ import (
 	restful "github.com/emicklei/go-restful"
 	"github.com/golang/glog"
 	"github.com/prometheus/client_golang/prometheus"
-	"gopkg.in/natefinch/lumberjack.v2"
+	lumberjack "gopkg.in/natefinch/lumberjack.v2" // goimports will remove this if the alias is not set
```

Comment:
whoa, that's interesting... I don't see that problem locally, but I might be using an older version of goimports.